rdbi-recipes/pom.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.lithium.dbi.rdbi</groupId>
+        <artifactId>rdbi-parent</artifactId>
+        <version>0.56-SNAPSHOT</version>
+        <relativePath>../rdbi-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>rdbi-recipes</artifactId>
+    <name>rDBI-recipes</name>
+    <url>https://github.com/lithiumtech/rdbi</url>
+    <description>
+        rDBI provides convenience utilites for Jedis, cleaning up some of Jedis's api pitfalls by automatically
+        returning the Jedis resource to the pool. It also has helper utilities for making Lua script calls to Redis easier.
+        rDBI library is inspired by the awesome library jDBI, a convenience library for SQL.
+    </description>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <build>
+        <plugins>
+            <!-- compiler -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.5.1</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
there's also a "sniffer" plugin (see bandolier) that will actively inspect code looking for java8 constructs that aren't allowed (this really applies to your rdbi-recipes-java7 module).  Otherwise, if you compile/build rdbi on a java8 JDK, it will still succeed even if you put lambdas/optional/instant or whatever in the java7 module.